package.json

```diff
@@ -112,6 +112,9 @@
       "@commitlint/config-conventional"
     ]
   },
+  "release": {
+    "branches": "['+([0-9])?(.{+([0-9]),x}).x', 'master', {name: 'next', channel: 'next', prerelease: 'next'}, 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}]"
```

Comment:
@tay1orjones does this `prerelease` for next need to be `true` instead of `next`?

Reply:
`prerelease: true` will automatically use the `name` in the version name, but I configured it with the expanded syntax so it's more clear as to what we've configured.

As an example:
```
{name: 'next', channel: 'next', prerelease: 'true'} would result in the format:
2.136.0-next.1

{name: 'next', channel: 'next', prerelease: 'next'} results in the format:
2.136.0-next.1

{name: 'next', channel: 'next', prerelease: 'dragons'} would result in the format:
2.136.0-dragons.1

{name: 'dragons', channel: 'next', prerelease: 'true'} would result in the format:
2.136.0-dragons.1
```